plugins/org.eclipse.n4js.dts/lib/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+!.gitignore
+!antlr4-runtime-4.7.2.jar
```

Comment:
It contains ANTLR4 that the generated parser/lexer uses. We cannot use antlr4 from Eclipse Orbit because (as far as I understand) that will impact dependency resolution of osgi/tycho (not maven). The result is that antlr3 is not used anymore in all the other bundles that require antlr3.